examples/all-clusters-app/nrfconnect/sysbuild/mcuboot/boards/nrf54l15dk_nrf54l15_cpuapp.conf

```diff
@@ -0,0 +1,41 @@
+#
+#    Copyright (c) 2025 Project CHIP Authors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+# TODO: Workaround, disable memory guard to avoid false faults in application after boot
+CONFIG_HW_STACK_PROTECTION=n
+
+CONFIG_BOOT_WATCHDOG_FEED=n
+
+# nRF54L15DK uses SPI NOR external flash
```

Comment:
You can skip adding here some configs, like CONFIG_PM_OVERRIDE_EXTERNAL_DRIVER_CHECK=y if you defined it in Kconfig.sysbuild. It may reduce maintenance costs here.